internal/featuresupport/features.go

```diff
@@ -308,21 +347,36 @@ func (feature *UserManagedNetworkingFeature) getIncompatibleArchitectures(_ *str
 	return nil
 }
 
+func (feature *UserManagedNetworkingFeature) isFeatureActive(cluster *common.Cluster, clusterUpdateParams *models.V2ClusterUpdateParams) bool {
+	if (swag.BoolValue(cluster.UserManagedNetworking) && clusterUpdateParams == nil) ||
```

Comment:
the condition is too long and it's hard to read it
please add a comment or split it